imap_processing/decom.py

```diff
@@ -0,0 +1,25 @@
+from space_packet_parser import parser, xtcedef
+
+
+def decom_packets(packet_file: str, xtce_packet_definition: str):
```

Comment:
```suggestion
def decom_packets(packet_file: Path, xtce_packet_definition: str):
```
Below you are calling `packet_file.open()`, but you can't do that on a string, so I think this can actually only be `Path` type right now. Alternatively you could change the below block to `with open(packet_file, "rb")` and that would handle both cases, either one seems fine just depends on what you want to do for typing purposes.

Reply:
Thank you for catching that.